dev/tools/grunt/tasks/static.js

```diff
@@ -17,7 +17,7 @@ module.exports = function (grunt) {
         };
 
     grunt.registerTask('static', function (target) {
-        var currentTarget = target || 'test',
+        var currentTarget = target || 'file',
```

Comment:
Ok, make sense. But change, please, in `dev/travis/script.sh` `grunt static` to `grunt static:test`.